src/instrument/utils/helper_functions.py

```diff
@@ -62,10 +62,22 @@ def is_notebook():
 
 def mpl_setup():
     """
-    MatPlotLib setup.
+    Matplotlib setup based on environment (Notebook or non-Notebook).
     """
     if not running_in_queueserver():
         import matplotlib.pyplot as plt
 
         if not is_notebook():
+            try:
+                # Try switching to 'tkagg' backend first
```

Comment:
I'm still not convinced we should use tkagg?  Will `bec` be able to use that correctly?

Reply:
[Asked](https://mattermost.hzdr.de/bluesky/pl/3pzyujymotfbxqkex3n9yf857r) on Mattermost

Reply:
@tacaswell says tkagg is not supported now

Reply:
I had assumed this was the case but thought that windows was supporting it hence trying that

Reply:
QtAgg should work everywhere.

Qt also plays much nicer with high-dpi screens.

Reply:
and tk will existing (almost) everywhere as the bindings are shipped with Python so this will almost always be the one you get.

Reply:
Fixed it, worked on my end because I do not use condo-forge and pip install everything. Made qtagg the default backend, and added PyQt5 to the toml, and removed PyQt=5 from the installation readme.  Using new install steps it should work